Baymax/DiagnosticsManager.swift

```diff
@@ -112,7 +112,14 @@ public class DiagnosticsManager {
         authenticationRequestClosure = authenticationHandler
         
         let swipeGesture = UISwipeGestureRecognizer(target: self, action: #selector(handleSwipeGesture))
-        swipeGesture.numberOfTouchesRequired = 4
+                
+        #if targetEnvironment(simulator)
+        let numberOfTouches = 2
```

Comment:
Won't this interfere with other things like zooming on a map?

Reply:
Shouldn't do, as that's a `UIPinchGestureRecognizer` :) Should be fairly easy to differentiate between the two :)

Reply:
Indeed, but some places (like websites) I think use 2 fingers, anyway no biggie :)